nipype/interfaces/afni/preprocess.py

```diff
@@ -2492,6 +2493,15 @@ class TProjectInputSpec(AFNICommandInputSpec):
                    even if -ort contains constant terms, as all means are
                    removed.""",
         argstr="-polort %d")
+    dsort = InputMultiObject(
+        File(
+            exists=True,
+            copyfile=False),
+        argstr="%s",
```

Comment:
you can change this to `argstr="-dsort %s..."` and it'll apply the same behavior you added in `_format_arg`

Reply:
@mattcieslak no worries, we love getting new contributions! the only thing missing is changing dsort's argstr (see above) and removing the `_format_arg` method